docs/rst/introduction/tools/shapes_demo.rst

```diff
@@ -3,5 +3,136 @@
+*   Ownership: The Ownership QoS determines whether the key (color) of a Topic (Shape) is owned by a single publisher.
+    If the selected ownership is ``EXCLUSIVE`` the publisher will use the Ownership strength value as the strength of its publication.
+    Only the publisher with the highest strength can publish in the same Topic with the same Key (see Fast DDS OwnershipQosPolicy Documentation).
+*   Deadline: The Deadline QoS determines the maximum expected amount of time between samples.
+    When the deadline is missed the application will be notified and a message will be printed on the console (see Fast DDS DeadlineQosPolicy Documentation).
+*   Lifespan: The Lifespan QoS determines the duration while the sample is still valid.
+    When a sample’s lifespan expires, it will be removed from publisher and subscriber histories (see Fast DDS LifespanQosPolicy Documentation).
+
+.. note::
+
+    Using Lifespan QoS will not have any visual effect.
+
+Subscribing to a Topic
+^^^^^^^^^^^^^^^^^^^^^^
+
+When the Subscriber button is pressed, a new window appear to allow the user to define the Shape (topic) and Quality of Service (QoS) for its subscription.
+The following image shows an example of the Subscribe menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/subscribe.png
+   :alt: Subscribe Window
+   :align: center
+
+This menu is highly similar to the Publication menu but the user cannot change the color and size of the Shape, and it has additional elements:
+
+*   Liveliness: This QoS policy is applied in the same way as in the publisher except for the Announcement Period, which does not apply for the Subscriber (see Fast DDS LivelinessQosPolicy Documentation).
+*   Time Based Filter: This value can be used by the user to specify the minimum amount of time (in milliseconds) that the subscriber wants between updates (see Fast DDS TimeBasedFilterQosPolicy Documentation).
+*   Content Based Filter: This filter draws a rectangle in the instances window.
+    Only the shapes that are included in this rectangle are accepted while the rest of them are ignored.
+    The user can dynamically resize and move this content filter.
+
+.. note::
+
+    Using Lifespan QoS will not have any visual effect.
+
+Participant configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The *ROS 2 Shapes Demo* application allows the user to define Participant policies.
+To see the Options window, please go to Options->Participant Configuration in the main bar.
+The following image shows the Options Menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/participant.png
+   :scale: 75 %
+   :alt: Options Window
+   :align: center
+
+*   Transport Protocol: You can select between UDP protocol, TCP LAN Server, TCP WAN Server or TCP Client, and Shared Memory protocol.
+
 .. note::
-    This page is under maintenance and will be updated soon.
+
+    This is under maintenance and will be updated soon.
```

Comment:
Is it?

Reply:
This was a note from Irene that I forgot to remove